### ISSUE TITLE
PHP error shows in update script

### ISSUE THREAD

**Issue by Skywalker-11** [it-novum/openITCOCKPIT#1020]
System: inplace upgrade openITCOCKPIT v3 (Debian jessie) to v4 (Debian buster) via apt
When upgrading to v4 there is an error logged but the update continues. After the upgrade is finished login works fine both for local and LDAP logins.

```
Deleted Aco node: /controllers/NagiosModule/Nagios and all children
Aco Update Complete
Checking permissions for user group "Administrator"...    Done
Checking permissions for user group "Viewer"...    Done
Restore default permissions for user group "Administrator"    Done
Re-Enable default permissions for user group "Viewer" but keep custom changes.    Done
---------------------------------------------------------------
Flush redis cache
OK
Notice Error: Undefined index: username
In [/opt/openitc/frontend/src/Command/Update3To4Command.php, line 247]

2020-07-29 14:14:25 Notice: Notice (8): Undefined index: username in [/opt/openitc/frontend/src/Command/Update3To4Command.php, line 247]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE/src/Error/BaseErrorHandler.php, line 188
App\Command\Update3To4Command::migrateEmailConfiguration() - APP/Command/Update3To4Command.php, line 247
App\Command\Update3To4Command::execute() - APP/Command/Update3To4Command.php, line 171
Cake\Console\BaseCommand::run() - CORE/src/Console/BaseCommand.php, line 179
Cake\Console\CommandRunner::runCommand() - CORE/src/Console/CommandRunner.php, line 336
Cake\Console\CommandRunner::run() - CORE/src/Console/CommandRunner.php, line 171
[main] - ROOT/bin/cake.php, line 12

Notice Error: Undefined index: password
In [/opt/openitc/frontend/src/Command/Update3To4Command.php, line 248]

2020-07-29 14:14:25 Notice: Notice (8): Undefined index: password in [/opt/openitc/frontend/src/Command/Update3To4Command.php, line 248]
Trace:
Cake\Error\BaseErrorHandler::handleError() - CORE/src/Error/BaseErrorHandler.php, line 188
App\Command\Update3To4Command::migrateEmailConfiguration() - APP/Command/Update3To4Command.php, line 248
App\Command\Update3To4Command::execute() - APP/Command/Update3To4Command.php, line 171
Cake\Console\BaseCommand::run() - CORE/src/Console/BaseCommand.php, line 179
Cake\Console\CommandRunner::runCommand() - CORE/src/Console/CommandRunner.php, line 336
Cake\Console\CommandRunner::run() - CORE/src/Console/CommandRunner.php, line 171
[main] - ROOT/bin/cake.php, line 12

Mail configuration "/opt/openitc/frontend/config/email.php" saved successfully.
Active users...    done
Create partition p_2020_29 in table statusengine_host_notifications   Ok
Create partition p_2020_30 in table statusengine_host_notifications   Ok
Create partition p_2020_31 in table statusengine_host_notifications   Ok
Create partition p_2020_32 in table statusengine_host_notifications   Ok
Migrating host notifications
3/3 [====================================================>] 100.00% 00:00:00
````

**Comment by nook24** [comment]
Maybe relates to #1016

**Comment by nook24** [comment]
This error message is from the migration of the email configuration. You should check if your system is able to send emails. If yes you can ignore this error. Otherwise please check the documentation about how to setup email notifications: https://openitcockpit.io/beginners/7-setup-email-notifications/

This will be fixed in 4.0.5 - thanks for reporting.
